package.json

```diff
@@ -15,8 +15,11 @@
     "copy-webpack-plugin": "^10.1.0",
     "css-loader": "^6.5.1",
     "css-minimizer-webpack-plugin": "^3.2.0",
+    "extract-loader": "^5.1.0",
+    "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.5.0",
-    "mini-css-extract-plugin": "^2.4.5",
+    "sass": "^1.45.0",
```

Comment:
something else I've never heard of... how many more meta-languages should I know about?

Reply:
You're in for a treat: https://roadmap.sh/frontend